go/kbfs/kbfsfuse/main.go

```diff
@@ -97,6 +97,9 @@ func start() *libfs.Error {
 			fuseLog, false /* superVerbose */)
 	}
 
+	logger.EnableBufferedLogging(loggingFrequency)
```

Comment:
this causes an error during compiling.

Reply:
`EnableBufferedLogging` does not take a `loggingFrequency`.